mc-to-mt-tp-conv/conv.sh

```diff
@@ -1,31 +1,31 @@
 #!/bin/bash
-cd "$(dirname "$0")"
-cd ..
+cd -- "$(dirname "$0")" || exit 1
+cd .. || exit 1
 rm -rf minetest-texture-pack
 rm -rf uctextures
 mkdir textures
 mkdir uctextures
-cd uctextures
+cd uctextures || exit 1
 mkdir b
 mkdir i
-cd ..
+cd .. || exit 1
 
 
-FILES=$(dirname "$0")/../assets/minecraft/textures/blocks/*.png
+FILES="$(dirname "$0")/../assets/minecraft/textures/blocks/*.png"
 for f in $FILES
 do
-	cp $f "uctextures/b/"
+	cp "$f" "uctextures/b/"
```

Comment:
I think `cp --` would be more safe because the output of `dirname "$0"` may begin with a dash.
```suggestion
	cp -- "$f" "uctextures/b/"
```